src/blocks/rsvp-response/edit.js

```diff
@@ -20,13 +23,39 @@ import EditCover from '../../components/EditCover';
  * @return {JSX.Element} The rendered React component.
  */
 const Edit = () => {
+	const isAdmin = wp.data.select('core').canUser('create', 'posts');
 	const blockProps = useBlockProps();
 
+	const [editMode, setEditMode] = useState(false);
+
+	const onEditClick = (e) => {
+		e.preventDefault();
+		setEditMode(!editMode);
+	};
+
 	return (
 		<div {...blockProps}>
-			<EditCover>
-				<RsvpResponse />
-			</EditCover>
+			{editMode && <RsvpResponseEdit />}
+			{!editMode && (
+				<EditCover>
+					<RsvpResponse />
+				</EditCover>
+			)}
+			{isAdmin && (
+				<BlockControls>
+					<ToolbarGroup>
+						<ToolbarButton
+							label="Edit Attendees"
```

Comment:
Change to just `Edit`. Make sure to add to translation function. `{__('Edit', 'gatherpress')}`. Super important we do this. We already have the plugin translated to 3 languages and we'll need to update when this feature goes out to ensure text is translated.